pkg/common/common.go

```diff
@@ -24,6 +24,15 @@ const (
 	// CDIOperatorName is the CDI operator name
 	CDIOperatorName = "cdi-operator"
 
+	// CDIControllerResourceName is the CDI controller resource name
+	CDIControllerResourceName = "cdi-deployment"
```

Comment:
this change actually rename the existing deployment from "cdi-controller" to "cdi-deplyment".

@yaroslavborbat is it on purpose? A typo? This change is not documented in the PR description.